blogs/urls.py

```diff
@@ -0,0 +1,25 @@
+from django.urls import path, re_path
+from rest_framework.urlpatterns import format_suffix_patterns
+from . import views
+
+from django.shortcuts import render
+
+ 
+app_name = 'blogs'
+
+urlpatterns = [
+    path('api/post_list/', views.BlogsList.as_view(), name='post_list'),
```

Comment:
It is good for now, but for the future go through this and try to follow naming conventions while creating REST API Endpoints. 
https://restfulapi.net/resource-naming/